metacat-testdata-provider/src/main/resources/metacatAllTypes.json

```diff
@@ -0,0 +1,925 @@
+{
```

Comment:
I assume these are the test stub data. Adding comments will be helpful in case we want to change something later. It looks to me that these are the canonical/source types with under a hive table.